src/exportJarSteps/GenerateJarExecutor.ts

```diff
@@ -138,7 +144,12 @@ export class GenerateJarExecutor implements IExportJarStepExecutor {
                             return;
                         }
                         for (const item of pickBox.selectedItems) {
-                            stepMetadata.elements.push(item.path);
+                            const classpath: IClassPath = {
+                                source: item.path,
+                                destination: undefined,
+                                isExtract: item.type === "external",
```

Comment:
The logic here is strange. It's hard to find any relationship between `external` and `isExtract`

Reply:
Will rename `isExtract` to `isDependency`